src/Features/Core/Portable/TodoComments/AbstractTodoCommentService.cs

```diff
@@ -38,32 +38,33 @@ public async Task<IList<TodoComment>> GetTodoCommentsAsync(Document document, IL
             // make sure given input is right one
             Contract.ThrowIfFalse(_workspace == document.Project.Solution.Workspace);
 
-            // same service run in both inproc and remote host, but remote host will not have RemoteHostClient service, 
-            // so inproc one will always run
-            var client = await document.Project.Solution.Workspace.TryGetRemoteHostClientAsync(cancellationToken).ConfigureAwait(false);
-            if (client != null && !document.IsOpen())
+            // run todo scanner on remote host. 
+            // we only run closed files to make open document to have better responsiveness. 
+            // also we cache everything related to open files anyway, no saving by running
+            // them in remote host
```

Comment:
comment seems odd.  there should still be a savings here by having less GC pressure in the VS process.  we should cosnider moving this out (in the future).

Reply:
This is just copied from before

Reply:
Sure, not in the scope of this PR though.